src/lib/prisma/users.ts

```diff
@@ -134,16 +116,20 @@ export async function getUsersEnrollsAndRequests(userId: string) {
           id: true,
         },
       },
-      requestedCourses: {
-        select: {
-          id: true,
-        },
-      },
     },
   });
   return courses;
 }
 
+export async function getUsersRequests(userId: string) {
+  const requests = await prisma.request.findMany({
+    where: {
+      userId: userId,
+    },
+  });
+  return requests;
+}
+
```

Comment:
Not a big thing but I think this function should be in the requests.ts file as the fetch is done from the request table

Reply:
The reason the function is in this file is that it only returns data from the users table, we talked about it and agreed that it makes sense to keep it here